cluster/examples/kubernetes/ceph/cluster-on-pvc.yaml

```diff
@@ -48,6 +48,10 @@ spec:
       # this needs to be set to false. For example, if using the local storage provisioner
       # this should be false.
       portable: true
+      # Certain storage class in the Cloud are slow
+      # Rook can configure the OSD running on PVC to accommodate that by tuning some of the Ceph internal
+      # Currently, "gp2" has been identified as such
+      tuneSlowDeviceClass: false
```

Comment:
This example yaml uses the `gp2` storage class so shall we set this to `true`?

Reply:
Yes!